.github/workflows/high-impact-alert.js

```diff
@@ -0,0 +1,28 @@
+const {
+  slackNotification,
+  getLocalConfigs,
+} = require('./helpers.js');
+
+const main = async (params) => {
+  const { context } = params;
+
+  try {
+    if (context.payload.label.name === 'high-impact') {
+      const { html_url, number, title } = context.payload.pull_request;
+      console.log('High impact label detected, sending Slack notification');
+      slackNotification(`:alert: High Impact PR has been opened: <${html_url}|#${number}: ${title}>.` +
+         ` Please prioritize testing the proposed changes.`, process.env.SLACK_HIGH_IMPACT_PR_WEBHOOK);
+    } else {
+      console.log('No high impact label detected');
+    }
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+if (process.env.LOCAL_RUN) {
+  const { context } = getLocalConfigs();
+  main({ context });
+}
```

Comment:
I'm curious, were the helpers to run things locally helpful?
I very much rely on developing locally first and only switching to CI when it's becoming necessary

Reply:
Initially, yes, but because I don't have a whole lot of experience with workflows, I ended up testing things on my fork and just updating the code there. In the future local development will likely be a more viable option.

Reply:
Just FYI u can also run the workflow on pushing updates, so working on a fork and triggering the workflow with new commits is also quite okay